connectivity/http/http.c

```diff
@@ -9,12 +9,16 @@
 #include "utils/macros.h"
 
 #define HTTP_LOGGER "http"
+#define MAX_REQUEST_LEN 65536
```

Comment:
What does `65536` stand for? Any reference?

Reply:
64K + 1 for NULL-terminated string.
We may discuss the maximum request length TA would like to support.

Reply:
Got it